_posts/2017-01-13-solidity-0.4.8-release-announcement.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.4.8 Release Announcement'
+date: '2017-01-13'
+author: Solidity Team
+category: Releases
+---
+
```

Comment:
Introduction sentence is missing here, too. I'm not sure if it's worth the effort adding one, I'm just pointing it out. :)